Js File/katas_02.js

```diff
@@ -0,0 +1,11 @@
+const peoples = [
+    { name: 'malika', age: 25 },
+    { name: 'chalana', age: 30 },
+    { name: 'eran', age: 35 },
```

Comment:
refactor this code, you can write click on the katas_02.js file and selected the format document option for do it